packages/perseus/src/widgets/grapher/util.tsx

```diff
@@ -26,11 +29,14 @@ const DEFAULT_BACKGROUND_IMAGE = {
     url: null,
 } as const;
 
-export const getEquationString = (props: any): string => {
-    const plot = props.plot;
+export const getEquationString = (plot: GrapherAnswerTypes): string => {
     if (plot.type && plot.coords) {
         const handler = GrapherUtil.functionForType(plot.type);
-        const result = handler.getEquationString(plot.coords, plot.asymptote);
+        const result = handler.getEquationString(
+            plot.coords,
+            // some graph types don't have this
+            (plot as any).asymptote,
+        );
```

Comment:
I'm not a fan of how this gets the equation string from the handler. (But! Still not as gross as interactive graph using strings to build the function names 😬) Just a qualm with the original code, not with your changes.

You don't have to do anything, I'm just making a note to myself that this needs to be updated whenever we try to merge grapher into interactive graph.